modules/administration/pages/kubernetes.adoc

```diff
@@ -126,25 +122,28 @@ The following steps will guide you through importing a previously deployed image
 
 The following steps will help you find additional runtime data.
 
-. Select to menu:Images[Images] , click the `Details` button on the right end of a row which has running instances.
-. Under the `Overview` tab, notice the data in `Runtime` and `Instances` fields under `Image Info` section.
+. Select menu:Images[Image List].
```

Comment:
You navigate to a menu item. This and the next step should be together:

. Navigate to menu:{Image List], locate the row that contains the running instance, and click btn:[Details].

(or something)

Reply:
ok, ready to be merged